### PR TITLE
Update ChangeLog for beta.9 release

### PR DESCRIPTION
**Pull request by daxian-dbw**
Update ChangeLog for beta.9 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,106 @@
...
+
+- Update Pi demo instructions about installing libunwind8. (#4974)
+- Add links on best practice guidelines in coding guideline. (#4983) (Thanks @iSazonov)
+- Reformat command line help for `powershell -Help` (#4989) (Thanks @iSazonov)
+- Change logo in readme to current black icon. (#5030)
+- Fix RPM package name in README.md (#5044)
+- Update `docs/building/linux.md` to reflect the current status of powershell build. (#5068) (Thanks @dee-see)
+- Add black version of `.icns` file for macOS. (#5073) (Thanks @thezim)
+- Update Arch Linux installation instructions. (#5048) (Thanks @kylesferrazza)
+- Add submodule reminder to testing-guidelines.md. (#5061) (Thanks @DdWr)
```
Comment (adityapatwardhan) [inline CHANGELOG.md]: ` around testing-guidelines.md

**Review by adityapatwardhan — APPROVED** [review]
left one minor comment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,106 @@
...
+- Update to use `pwsh` in macOS launcher. (#5174) (Thanks @thezim)
+- Add code to send web hook for Travis-CI daily build. (#5183)
+- Add `global.json` to pick correct SDK version. (#5118) (Thanks @rkeithhill)
+- Update packaging to only package PowerShell binaries when packaging symbols. (#5145)
+- Update docker files and related due to the name change. (#5156)
+
+### Code Cleanup
+
+- Clean up Json cmdlets. (#5001) (Thanks @iSazonov)
+- Remove some code that is ancient and has never been used. (#5066)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Too generic, how about:

```none
Removed code that has never been used: RELATIONSHIP_SUPPORTED and SUPPORTS_IMULTIVALUEPROPERTYCMDLETPROVIDER
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,106 @@
...
+- Remove PSMI code that has never been used. (#5075)
+- Remove unreachable code for `Stop-Job`. (#5091) (Thanks @travisty-)
+
+### Test
+
+- Fix Delay Test Race Condition in `WebCmdlets.Tests.ps1`. (#5035) (Thanks @markekraus)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Rewords as

```none
Fix race condition between test WebListener and WebCmdlets tests
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,106 @@
...
+- Add warning to `Start-PSPester` if Pester module is not found (#5069) (Thanks @DdWr)
+- Add tests for DSC configuration compilation on Windows. (#5011)
+- Test fixes and code coverage automation fixes. (#5046)
+
+### Documentation and Help Content
+
+- Update Pi demo instructions about installing libunwind8. (#4974)
+- Add links on best practice guidelines in coding guideline. (#4983) (Thanks @iSazonov)
+- Reformat command line help for `powershell -Help` (#4989) (Thanks @iSazonov)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Looks weird with H capitalized on `-Help`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,106 @@
...
+- Change logo in readme to current black icon. (#5030)
+- Fix RPM package name in README.md (#5044)
+- Update `docs/building/linux.md` to reflect the current status of powershell build. (#5068) (Thanks @dee-see)
+- Add black version of `.icns` file for macOS. (#5073) (Thanks @thezim)
+- Update Arch Linux installation instructions. (#5048) (Thanks @kylesferrazza)
+- Add submodule reminder to testing-guidelines.md. (#5061) (Thanks @DdWr)
+- Update instructions in `docs/building/internals.md` for building from source. (#5072) (Thanks @kylesferrazza)
+- Add user voice link to Issue Template. (#5100) (Thanks @markekraus)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Should be: UserVoice

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Comment by daxian-dbw** [comment]
@SteveL-MSFT I have addressed your comments. Please take another look, thanks!

**Review by SteveL-MSFT — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,106 @@
...
+- Remove `-ComputerName` from `Get/Set/Remove-Service`. (#5094)
+- Rename the executable name from `powershell` to `pwsh`. (#5101)
+- Remove `RunspaceConfiguration` support. (#4942)
+- Remove `-ComputerName` support since .NET Core `Process.GetProcesses(computer)` returns local processes. (#4960)
+- Make `-NoTypeInformation` the default on `Export-Csv` and `ConvertTo-Csv`. (#5164) (Thanks @markekraus)
+- Unify cmdlets with parameter `-Encoding` to be of type `System.Text.Encoding`. (#5080)
+
+### Engine updates and fixes
+
+- Fix powershell to update the `PATH` environment variable only if `PATH` exists. (#5021)
```
Comment (mirichmo) [inline CHANGELOG.md]: Capitalize PowerShell

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,106 @@
...
+
+### Build and Packaging Improvements
+
+- Treat `.rtf` files as binary so EOL don't get changed. (#5020)
+- Improve the output of `tools/installpsh-osx.sh` and update Travis-CI to use Ruby 2.3.3. (#5065)
+- Improve `Start-PSBootstrap` to locate dotnet SDK before installing it. (#5059) (Thanks @PetSerAl)
+- Fix the prerequisite check of the MSI package. (#5070)
+- Support creating `tar.gz` package for Linux and macOS. (#5085)
+- Add release builds that produce symbols for compliance scans. (#5086)
+- Update existing docker files for the Linux package changes. (#5102)
```
Comment (mirichmo) [inline CHANGELOG.md]: Capitalize Docker

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,106 @@
...
+- Fix RPM package name in `README.md`. (#5044)
+- Update `docs/building/linux.md` to reflect the current status of powershell build. (#5068) (Thanks @dee-see)
+- Add black version of `.icns` file for macOS. (#5073) (Thanks @thezim)
+- Update Arch Linux installation instructions. (#5048) (Thanks @kylesferrazza)
+- Add submodule reminder to `testing-guidelines.md`. (#5061) (Thanks @DdWr)
+- Update instructions in `docs/building/internals.md` for building from source. (#5072) (Thanks @kylesferrazza)
+- Add UserVoice link to Issue Template. (#5100) (Thanks @markekraus)
+- Add `Get-WebListenerUrl` Based Examples to WebListener `README.md`. (#4981) (Thanks @markekraus)
+- Add document about how to create cmdlet with dotnet CLI. (#5117) (Thanks @rkeithhill)
+- Update the help text for powershell with the new name `pwsh`. (#5182)
```
Comment (mirichmo) [inline CHANGELOG.md]: Capitalize PowerShell? or `powershell`?

**Review by mirichmo — CHANGES_REQUESTED** [review]
Just a few minor things

**Comment by daxian-dbw** [comment]
@mirichmo You comments have been addressed. Please take another look.

**Review by mirichmo — APPROVED** [review]

**Pull request merged**